hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/freon/TestOMSnapshotDAG.java

```diff
@@ -284,6 +284,7 @@ public void testDAGReconstruction()
     snapDB1.close();
     snapDB2.close();
     snapDB3.close();
+    ozoneManager.getOmSnapshotManager().getSnapshotCache().invalidateAll();
```

Comment:
Why do you need to explicitly invalidate the cache? OM stop should do that for you when you restart the cluster in next line.

 [OzoneManager#stop()](https://github.com/apache/ozone/blob/e057eed29fe320685114670ceb7620d9367e0baa/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java#L2194) calls [omSnapshotManager.close()](https://github.com/apache/ozone/blob/e057eed29fe320685114670ceb7620d9367e0baa/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java#L2235) which [invalidates the cache](https://github.com/apache/ozone/blob/c4d9456120868aedf0635aac6e6d82b92a0bbe37/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java#L961).
 
cluster.restartOzoneManager() -> ozoneManager.stop() -> omSnapshotManager.close() -> snapshotCache.invalidateAll().